test/automated/ansible/roles/create-inventory/defaults/main.yml

```diff
@@ -3,5 +3,6 @@
 ec2_private_key_file: "~/.ssh/caos-dev-arm.cer"
 output_inventory: "inventory.ec2"
 instance_prefix: ""
```

Comment:
`ansible_password_windows` will be passed form env var and it will be stored in the inventory file after provisioning ec2 instances and before running harvest/packaging tests